companies/two.json

```
@@ -0,0 +1,19 @@
+{
+    "slug": "two",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "utility"
+    ],
+    "name": "T.W.O. Technische Werke Osning GmbH",
+    "address": "Hopfengarten 10\n33775 Versmold\nDeutschland",
```

As this is the address of the external DPO, we need to mention that company's name to make sure letters actually arrive.

```suggestion
    "address": "c/o EDV-Unternehmensberatung Floß GmbH\nHopfengarten 10\n33775 Versmold\nDeutschland",
```